build.sbt

```diff
@@ -232,6 +260,10 @@ val commonSettings = Def
         Seq(Tests.Argument(TestFrameworks.ScalaTest, "-l", "org.scalatest.tags.Slow"))
       }
     },
+    unusedCompileDependenciesFilter -= Seq(
+      moduleFilter("org.scala-lang", "scala-reflect"),
+      moduleFilter("org.scala-lang.modules", "scala-collection-compat")
```

Comment:
Those libs are required for scala 2.12 and considered unused for 2.13. Silence them